### PR TITLE
Update for droiddriver 0.9.1

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

**Comment by bootstraponline** [comment]
@ddkjin I've updated the [espresso basicsample](https://github.com/googlesamples/android-testing/tree/master/espresso/BasicSample) to use the droiddriver 0.9.1 release. Let me know if this is the correct way to use droiddriver. The goal is to have a simple Espresso integration example for people to look at. After this, I'll look at a standalone DD example.

I noticed two areas for enhancement:
1. [resourceId ints](https://github.com/appium/droiddriver/issues/27)
2. [proper keyboard closing instead of using the back button](https://github.com/appium/droiddriver/issues/8)

Thanks.

**Comment by bootstraponline** [comment]
@ddkjin What do you think now?

File: espresso_sample/espresso/BasicSample/app/src/androidTest/java/com/example/android/testing/espresso/BasicSample/DDTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        ActivityUtils.setRunningActivitySupplier(new ActivityUtils.Supplier<Activity>() {
+            @Override
+            public Activity get() {
+                return Iterables.getFirst(ActivityLifecycleMonitorRegistry.getInstance().getActivitiesInStage(Stage.RESUMED), null);
```
Comment (ddkjin) [inline espresso_sample/espresso/BasicSample/app/src/androidTest/java/com/example/android/testing/espresso/BasicSample/DDTest.java:36]: So you could use an 'internal' package? Sweet ;)

Comment (bootstraponline) [inline espresso_sample/espresso/BasicSample/app/src/androidTest/java/com/example/android/testing/espresso/BasicSample/DDTest.java:36]: yes :smile:

**Comment by ddkjin** [comment]
LGTM, but I'm not familiar with the UI - do I need to +1?

**Comment by bootstraponline** [comment]
> LGTM, but I'm not familiar with the UI - do I need to +1?

To add a +1, that's just done with a comment or using the emoji :+1: . GitHub doesn't have a formal system for tracking approvals like gerrit. Once approved, then the green merge button is pressed and the changes are merged into master.

merge

Since this looks okay, then I'll merge. Thanks for reviewing!

**Pull request merged**